### PR TITLE
Update fields in source-connectors specifications: drift, harvest, hubspot, intercom, monday, clickhouse, bing-ads, snapchat-marketing, microsoft-teams

### LINKED ISSUE #8361: Update fields in source-connectors specifications: drift, harvest, hubspot, intercom, notion, monday, microsoft-teams, bing-ads, snapchat-marketing
Add title and description properties to each field in source-connectors specifications:
source-drift
source-harvest
source-hubspot
source-intercom
source-snapchat-marketing
source-bing-ads
source-microsoft-teams
source-monday
**source-notion** - _tests do not work, will be processed separately_

[EPIC] Add title and description properties to each field in connectors specifications and order input fields appropriately #8324

### PR DESCRIPTION
**Pull request by ycherniaiev**
## What
Add title and description properties to each field in source-connectors specifications:
source-drift
source-harvest
source-hubspot
source-intercom
source-clickhouse
source-snapchat-marketing
source-bing-ads
source-microsoft-teams
source-monday

### CODE REVIEW

File: airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
               },
               "ids": {
                 "type": "array",
+				"title": "IDs",
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json:54]: still tabs ?

File: airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json [hunk 2]
```diff
@@ -64,37 +65,44 @@
       },
       "client_id": {
         "type": "string",
-        "description": "ID of your Microsoft Advertising client application.",
+		"title": "Client ID",
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json:68]: stil tabs

File: airbyte-integrations/connectors/source-intercom/source_intercom/spec.json [hunk 3]
```diff
@@ -9,14 +9,15 @@
...
+		"title": "Start Date",
+        "description": "UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated.",
         "examples": ["2020-11-16T00:00:00Z"],
         "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
       },
       "access_token": {
         "title": "Access Token",
         "type": "string",
-        "description": "Access token generated either from an oauth flow or from the Intercom Developer dashboard. See the <a href=\"https://docs.airbyte.io/integrations/sources/intercom\">docs</a> for more information on how to obtain this key manually.",
+        "description": "Access Token for making authenticated requests.">docs</a> for more information on how to obtain this key manually.",
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-intercom/source_intercom/spec.json:20]: something really broken here

**Review by grubberr — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.json [hunk 4]
```diff
@@ -75,7 +75,7 @@
                 "order": 0
               },
               "api_key": {
-                "title": "API key",
+              "title": "API key",
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.json:78]: 'title' need to move one space right ?

File: airbyte-integrations/connectors/source-intercom/source_intercom/spec.json [hunk 5]
```diff
@@ -6,17 +6,18 @@
     "type": "object",
     "required": ["start_date", "access_token"],
     "additionalProperties": true,
-    "properties": {
+    "properties":
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-intercom/source_intercom/spec.json:9]: lost '{'

File: airbyte-integrations/connectors/source-monday/source_monday/spec.json [hunk 6]
```diff
@@ -9,7 +9,8 @@
     "properties": {
       "api_token": {
         "type": "string",
-        "description": "This is the API token to authenticate requests to Monday. Profile picture (bottom left) => Admin => API",
+        "title": "Personal Access Token"
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-monday/source_monday/spec.json:12]: lost comma

**Review by grubberr — CHANGES_REQUESTED** [review]

**Review by grubberr — APPROVED** [review]

**Review by nataliekwong — APPROVED** [review]

File: airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json [hunk 7]
```diff
@@ -64,37 +65,44 @@
...
         "type": "string",
-        "description": "The long-lived Refresh token received via grant_type=refresh_token request.",
+        "title": "Refresh Token",
+        "description": "Refresh Token to renew the expired Access Token.",
         "airbyte_secret": true
       },
       "user_id": {
         "type": "string",
+        "title": "User ID",
         "description": "Unique user identifier."
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json:98]: what is the difference between this and customer ID? where can the user obtain this ID?

File: airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json [hunk 8]
```diff
@@ -64,37 +65,44 @@
...
         "type": "string",
-        "description": "Secret of your Microsoft Advertising client application.",
+        "title": "Client Secret",
+        "description": "The Client Secret of your Microsoft Advertising developer application.",
         "airbyte_secret": true
       },
       "customer_id": {
         "type": "string",
-        "description": "User's customer ID."
+        "title": "Customer ID",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json:80]: where can the user obtain this ID?

File: airbyte-integrations/connectors/source-microsoft-teams/source_microsoft_teams/spec.json [hunk 9]
```diff
@@ -10,21 +10,22 @@
       "tenant_id": {
         "title": "Directory (tenant) ID",
         "type": "string",
-        "description": "Directory (tenant) ID"
+        "description": "A globally unique identifier (GUID) that is different than your organization name or domain."
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-microsoft-teams/source_microsoft_teams/spec.json:13]: this makes it sound like it's a random GUID? where can the user obtain this?

File: airbyte-integrations/connectors/source-notion/source_notion/spec.json [hunk 10]
```diff
@@ -1,5 +1,5 @@
 {
-  "documentationUrl": "https://docsurl.com",
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/notionl.com",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-notion/source_notion/spec.json:2]:
```suggestion
  "documentationUrl": "https://docs.airbyte.io/integrations/sources/notion",
```

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json (hunk 7 above)
Comment (ycherniaiev) [reply airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json:98]: It seems like user ID is Account ID. But not quite sure

**Comment by ycherniaiev** [comment]
@sherifnada Corrections are made. Approve please

**Review by sherifnada — APPROVED** [review]

**Review by igrankova — APPROVED** [review]

**Pull request merged**